pdfly/cli.py

```diff
@@ -54,8 +54,6 @@ def extract_images(
         typer.Argument(
             exists=True,
             file_okay=True,
-            dir_okay=False,
```

Comment:
https://github.com/py-pdf/pdfly/blob/main/pdfly/extract_images.py#L14 - that wouldn't work with a directory, would it?

Typer seems not to define a default for dir_okay: https://typer.tiangolo.com/tutorial/parameter-types/path/#path-validations
But it refers to click, where the default is True.

Reply:
Oh well spotted 👍

Yes, it seems that the default is `dir_okay=True`:
https://github.com/fastapi/typer/blob/master/typer/params.py#L301

I misunderstood the role of this flag, I thought it meant "the parent directory must exist" 😅

I will edit this PR & preserve this flag.